SoftLayer/CLI/object_storage/credential/delete.py

```diff
@@ -0,0 +1,22 @@
+"""Delete the credential of an Object Storage Account."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--credential_id', '-id', type=click.INT,
+              help="This is the credential id associated with the volume")
+@environment.pass_env
+def cli(env, identifier, credential_id):
+    """Delete the credential of an Object Storage Account."""
+
+    mgr = SoftLayer.ObjectStorageManager(env.client)
+    credential = mgr.delete_credential(identifier, credential_id=credential_id)
+
+    if credential:
```

Comment:
I don't think the `if` here is needed, since `mgr.delete_credentials` will likely just throw an exception if anything goes wrong. Or add an `else` to print out of the command returned False for some reason.

Reply:
Done.